prisma/base/eventStore.prisma

```diff
@@ -1,6 +1,6 @@
 datasource db {
   provider = "mongodb"
-  url      = env("EVENT_STORE_DB_URL")
+  url      = "mongodb+srv://test:test@localhost/test"
```

Comment:
Why hardcode?

Is this for local teesting only?

Reply:
These values are just default values and we are overriding them from code. So instead of having an empty string, I just specified the values as test.